ParquetClassLibrary/Sandbox/Parquets/AllParquets.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.Collections.Generic;
+using ParquetClassLibrary.Sandbox.ID;
+
+namespace ParquetClassLibrary.Sandbox.Parquets
+{
+    /// <summary>
+    /// Stores all defined parquet models for use in sandbox-mode.
+    /// This collection is the source of truth about parquets for the rest of the library,
+    /// something like a color palette that other classes can paint with.
+    /// </summary>
+    public static class AllParquets
+    {
+        #region Test Values
+        /// <summary>Used in test patterns in QA routines.</summary>
+        public static readonly Floor TestFloor = new Floor(-Assembly.FloorIDs.Minimum, "Grass Floor Test Parquet");
+
+        /// <summary>Used in test patterns in QA routines.</summary>
+        public static readonly Block TestBlock = new Block(-Assembly.BlockIDs.Minimum, "Brick Block Test Parquet");
+
+        /// <summary>Used in test patterns in QA routines.</summary>
+        public static readonly Furnishing TestFurnishing = new Furnishing(-Assembly.FurnishingIDs.Minimum, "Chair Furnishing Test Parquet");
+
+        /// <summary>Used in test patterns in QA routines.</summary>
+        public static readonly Collectable TestCollectable = new Collectable(-Assembly.CollectableIDs.Minimum, "Flowers Collectable Test Parquet");
+        #endregion
+
+        /// <summary>A collection of all defined parquets of all subtypes.</summary>
+        private static readonly Dictionary<ParquetID, ParquetParent> _table = new Dictionary<ParquetID, ParquetParent>
```

Comment:
nit: _table is a pretty weak name.

Panda: I would probably name it "by blank", like "_parquetByParquetID" or "_parquet_byParquetID."
Maussy: I'd probably go with "_parquetLookupTable", something that describes its contents.

Reply:
Good call!  Fixed in e0f4be0